ui/components/multichain/import-token-link/import-token-link.tsx

```diff
@@ -41,18 +45,21 @@ export const ImportTokenLink = ({ className, ...props }) => {
     >
```

Comment:
We were experiencing typing issues with spreading props in the component-library components because of how the polymorphic prop typing works. You may need to update line `44` {...props} to
```
{...(props as BoxProps<'div'>)}
```
This is casting the `...props` variable to type, `BoxProps`(`margin`, `display`, etc). The `<'div'>` is a generic type which we are passing to the polymorphic type and is saying to accept all html `div` attributes.

Reply:
Thanks for explaining it so nicely George! Really helpful. Updated here https://github.com/MetaMask/metamask-extension/pull/20474/commits/3cd657a4ee462b945aa54b5cfb172453751d4f7e